JustTrack/UnitTests/JEEventInternalTests.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  JEEventInternalTests.swift
+//  JustTrack
+//
+//  Created by Richard Rauser [Contractor] on 01/10/2018.
```

Comment:
Can we change this to match the rest of _JustTrack_? Especially the _Contractor_ bit? :) @albertodebortoli-je @richardrauser-je

Reply:
@joakimlazakis-je thanks for noticing, I'll remove it.

Reply:
Ha, of course. :D